emoji-ui/src/components/emojiItem.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import './emojiItem.css';
+
+class EmojiItem extends Component {
+  static propTypes = {
+    isSelected: PropTypes.bool.isRequired,
+    url: PropTypes.string.isRequired,
+    clickHandler: PropTypes.func.isRequired.isRequired,
+  };
+
+  render() {
+    const { isSelected, url, clickHandler } = this.props;
+    const styles = { backgroundImage: `url(${url})` };
```

Comment:
🙀  is this better than `img` tags?

Reply:
it makes it super easy to center the images and display them at the same size.

Reply:
good question! i am not sure, i do it this way because we do this in the merchant app and i always wonder why.

Reply:
Long answer: you can center and size image tags with flexbox if you only want the equivalent of `background-size: contain`, even when dealing with images of unknown dimensions. But this is a lot simpler.

And if you want `background-size: cover`, with images of unknown (up front) dimensions, you're probably looking at js if you use image tags.

Reply:
oh, i forgot -- if the images are taller than they are wide, they might start popping out even with flexbox and overflow:hidden. i would have to play around to figure out what to do with that... seems like unnecessary work most of the time though.